README.md

```diff
@@ -1 +1,90 @@
-# rails_budget_app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Budget App
+
+> Budget app is an app that helps you enter your planned expenses in categories
+
+# The App Deployment
+_[Here](https://www.loom.com/share/2493da8e15e14088827bc8d9ca24d71c)  The presentation video link_
+_[Live](https://verdant-torte-ebb866.netlify.app) The Live Application on Netlify
```

Comment:
Greate implementation of the project requirements

- [ ] Please fix your deployment link is broken